src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -519,6 +519,7 @@ frmelmnts.lbl.chapter = Chapter
 frmelmnts.lbl.charError = Password must contain at least 8 alphanumeric characters.
 frmelmnts.lbl.characterleft = Characters remaining
 frmelmnts.lbl.chartSummary = Chart summary
+frmelmnts.lbl.updateChartSummary = Update Chart summary
```

Comment:
Please check the case. Please capitalize the "s" in summary as well..

Reply:
resolved